README.md

```diff
@@ -86,6 +86,7 @@ contra-env-setup/playbooks/group_vars/all/global.yml
 * memory: Memory size to use for the VM : default=6400mb
 * cpus: Number of cpus to use for minishift VM: default=2
 * minishift_iso: ISO image to use : default=http://artifacts.ci.centos.org/fedora-atomic/minishift/iso/minishift.iso 
+* minishift_insecure_registry: Additional insecure registries : default="" (not used)
```

Comment:
Should we call this `minishift_insecure_registries` since it can contain multiple registries?

Reply:
I'm fine even with `minishift_insecure_registries` I used singular because of singular in minishift config.
`minishift config set insecure-registry`

Reply:
OK, I wasn't aware of that. Let's keep the singular then.